src/app/clientPage.tsx

```diff
@@ -78,7 +78,7 @@ export default function ClientPage({ searchParams }: { searchParams: { [key: str
   ) => {
     let validURL = inputUrl;
     // Readability requires us to send urls with the correct format, but because we want to support more forms of urls (google.com || www.google.com || https://www.google.com etc) we need to append the protocol before we send the data off.
-    if (inputUrl && !/(https:\/\/) | (http:\/\/)/i.test(inputUrl)) {
+    if (inputUrl && !/(https:\/\/)|(http:\/\/)/i.test(inputUrl)) {
```

Comment:
didn't  know about this 😄

Reply:
it's within the regex, that's a common mistake of mine 😅